src/main/flight/pid.c

```diff
@@ -1021,3 +1067,16 @@ void pidSetAcroTrainerState(bool newState)
     }
 }
 #endif // USE_ACRO_TRAINER
+
+void pidSetAntiGravityState(bool newState)
+{
+    if (newState != antiGravityEnabled) { // reset the accelerator on state changes
+        itermAccelerator = 1.0f;
+    }
+    antiGravityEnabled = newState;
+}
+
+bool pidAntiGravityEnabled(void)
+{
+    return antiGravityEnabled;
+}
```

Comment:
No newline at the end of file

Reply:
fixed, put one at the end of mixer.c also